tests/e2e/e2e_ibc_test.go

```diff
@@ -98,18 +98,31 @@ func (s *IntegrationTestSuite) hermesClearPacket(configPath, chainID, channelID
 	}
 
 	stdout, stderr := s.executeHermesCommand(ctx, hermesCmd)
-	if strings.Contains(stdout, "ERROR") || strings.Contains(stderr, "ERROR") {
+	if strings.Contains(string(stdout), "ERROR") || strings.Contains(string(stderr), "ERROR") {
 		return false
 	}
 
 	return true
 }
 
+type Response struct {
```

Comment:
Would rename this type to sthg more self-explantory like `RelayerPacketsOutput`.